### ISSUE TITLE
Centre='variables(/trials)' causes error

### ISSUE THREAD

**Issue by PhysXu** [hyperspy/hyperspy#2206]
When I apply centre='variables', the spectrum extracted from PCA_SI and SI in the same pixel is quite strange. Centre='trials' is the same.
```
SI.decomposition(normalize_poissonian_noise=True, centre='variables')
PCA_SI = SI.get_decomposition_model(8)
(SI + PCA_SI* 1j).plot()
```
![2](https://user-images.githubusercontent.com/35793078/59240206-395c8c80-8c36-11e9-89f0-794b192ceb48.png)

![1](https://user-images.githubusercontent.com/35793078/59240183-20ec7200-8c36-11e9-9807-a5eca4737c9f.png)

Without **centre**
![图片](https://user-images.githubusercontent.com/35793078/59242391-ee467780-8c3d-11e9-88ed-d0f927328fb8.png)

With **centre='trials'**
![图片](https://user-images.githubusercontent.com/35793078/59242402-fc949380-8c3d-11e9-8c6b-21971517b179.png)

**Comment by francisco-dlp** [comment]
Yes, ``centre`` and ``normalize_poisson_noise`` are not compatible. That should raise an error.

**Comment by ericpre** [comment]
Fixed in #2321.
